lhotse/features/opensmile.py

```diff
@@ -0,0 +1,180 @@
+from dataclasses import asdict, dataclass
+from typing import Any, Dict, List, Union, Optional, Sequence
+
+import numpy as np
+import warnings
+from lhotse.features.base import FeatureExtractor, register_extractor
+from lhotse.utils import is_module_available, Seconds, compute_num_frames
+
+
+@dataclass
+class OpenSmileConfig:
+    """
+    OpenSmile configs are stored in separated txt files in its specific format.
+    You can specify predefined config by setting ``feature_set`` and ``feature_level``
+    class attributes with:
+    (1) ``FeatureSet`` and ``FeatureLevel`` classes predefined in
+    https://github.com/audeering/opensmile-python/blob/master/opensmile/core/define.py
+    OR
+    (2) strings refered to enum members,
+    In opensmile-python You can also create your own config file and pass its path and
+    corresponding feature level as documented here
+    https://audeering.github.io/opensmile-python/usage.html#custom-config.
+    For now custom configs are not supported in this extractor.
+    """
+
+    feature_set: Union[str, Any] = "ComParE_2016"  # default feature set or
+    # string with set name
+    feature_level: Union[str, Any] = "lld"  # default feature level or level name
+    options: dict = None  # dictionary with optional script parameters
+    loglevel: int = 2  # log level (0-5), the higher the number the more log
+    # messages are given
+    logfile: str = None  # if not ``None`` log messages will be stored to this file
```

Comment:
```suggestion
    logfile: Optional[str] = None  # if not ``None`` log messages will be stored to this file
```
And also for other fields with default value `None`.